src/commands/dev/dev.ts

```diff
@@ -2,6 +2,7 @@ import process from 'process'
 
 import type { NetlifyAPI } from '@netlify/api'
 import { applyMutations } from '@netlify/config'
+import { ensureNetlifyIgnore } from '@netlify/dev-utils'
```

Comment:
I know this is a draft! but some pointers: we don't want to lose [these logs](https://github.com/netlify/primitives/blob/578e5f27148c3283aadf29b5b99147d297b43b06/packages/dev-utils/src/lib/gitignore.ts#L40-L41), so we'll need to pass in a logger.

it might also be nice to do this in a util (maybe keep `src/utils/gitignore.ts`) so we don't need to think about this everywhere we use it, especially because I think it will need to be a little awkward, something like:

```js
import { ensureNetlifyIgnore as originalEnsureNetlifyIgnore } from '@netlify/dev-utils'
import {log, warn, logError} from './command-helpers.js'

export const ensureNetlifyIgnore = async (rootDir: string) => originalEnsureNetlifyIgnore(
  rootDir,
  {log, warn, error: logError},
)
```